roles/apache/templates/common.conf.j2

```diff
@@ -33,6 +33,7 @@ SetEnvIf Referer "^https://{{ georchestra.fqdn|replace('.','\.') }}/" mysdi
 <Location /geoserver>
   Header set Access-Control-Allow-Origin "*"
   Header set Access-Control-Allow-Headers "Origin, X-Requested-With, Content-Type, Accept"
+  RequestHeader set "X-Forwarded-Proto" expr=%{REQUEST_SCHEME}
```

Comment:
Maybe using `RequestHeader set "X-Forwarded-Proto" "https"` instead ? This would be coherent with https://github.com/georchestra/georchestra/pull/3895 anyway